listenbrainz/webserver/static/js/src/recommendations/RecommendationCard.tsx

```diff
@@ -1,96 +1,275 @@
 import * as React from "react";
+import {
+  faAngry,
+  faFrown,
+  faMeh,
+  faSmileBeam,
+  faGrinStars,
+} from "@fortawesome/free-solid-svg-icons";
+
+import {
+  faThumbsUp as faThumbsUpRegular,
+  faAngry as faAngryRegular,
+  faFrown as faFrownRegular,
+  faMeh as faMehRegular,
+  faSmileBeam as faSmileBeamRegular,
+  faGrinStars as faGrinStarsRegular,
+} from "@fortawesome/free-regular-svg-icons";
+
 import { get as _get } from "lodash";
-import { faEllipsisV } from "@fortawesome/free-solid-svg-icons";
 import { IconProp } from "@fortawesome/fontawesome-svg-core";
+import { IconDefinition } from "@fortawesome/fontawesome-common-types";
```

Comment:
This fails even though the same import is used in RecommendationControl here 

```
> listenbrainz@1.0.0 format /code
> eslint ./static/js/src --ignore-path .gitignore --ext .js,jsx,ts,tsx --fix --quiet


/code/static/js/src/recommendations/RecommendationCard.tsx
  21:32  error  Unable to resolve path to module '@fortawesome/fontawesome-common-types'  import/no-unresolved

✖ 1 problem (1 error, 0 warnings)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! listenbrainz@1.0.0 format: `eslint ./static/js/src --ignore-path .gitignore --ext .js,jsx,ts,tsx --fix --quiet`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the listenbrainz@1.0.0 format script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-11-05T15_32_35_543Z-debug.log
```

Reply:
If you look more closely at the code line you linked, you'll notice it has a `// eslint-disable-line import/no-unresolved` commet which instructs eslint to ignore this issue.
That's why it's working there, at least, although I can't tell you what the actual issue is
Considering everything is working and the types are only used for development/compilation, I would just slap the same comment as in the other file and be done with it :)

Reply:
Oooo.... I didn't know comments can be that useful ;)